support/@fractal/polkadot-utils/examples/high_throughput_tranfers.js

```diff
@@ -0,0 +1,66 @@
+const {ApiPromise, Keyring, WsProvider} = require('@polkadot/api');
+const {TxnBatcher} = require('../build/main');
+
+async function isolatedBatcher() {
+  const provider = new WsProvider('ws://localhost:9944');
+  const api = await ApiPromise.create({provider});
```

Comment:
We only need to if we ever try to use those types. Since this is a generic library, it should be usable against any Substrate runtime. We still get a warning when running locally, but it doesn't prevent anything from working.